packages/front-end/pages/admin/OrganizationRow.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from "react";
+import { OrganizationInterface } from "@back-end/types/organization";
+import { UserInterface } from "@back-end/types/user";
+import clsx from "clsx";
+import {
+  FaAngleDown,
+  FaAngleRight,
+  FaPencilAlt,
+  FaTrash,
+} from "react-icons/fa";
+import { date } from "shared/dates";
+import stringify from "json-stringify-pretty-compact";
+import Collapsible from "react-collapsible";
+import useApi from "@/hooks/useApi";
+import { useLocalStorage } from "@/hooks/useLocalStorage";
+import EditOrganization from "@/components/Admin/EditOrganization";
+import Code from "@/components/SyntaxHighlighting/Code";
+import DeleteOrganizationModal from "@/components/Admin/DeleteOrganizationModal";
+import MembersTable from "./MembersTable";
+
+export default function OrganizationRow({
+  organization,
+  current,
+  switchTo,
+  showExternalId,
+  onUpdate,
+}: {
+  organization: OrganizationInterface;
+  switchTo: (organization: OrganizationInterface) => void;
+  current: boolean;
+  showExternalId: boolean;
+  onUpdate: () => void;
+}) {
+  const [expanded, setExpanded] = useState(false);
+  const [editOrgModalOpen, setEditOrgModalOpen] = useState(false);
+  const [deleteOrgModalOpen, setDeleteOrgModalOpen] = useState(false);
+  const [lastPickedOrg, setLastPickedOrg] = useLocalStorage(
+    "gb-last-picked-org",
+    ""
+  );
+
+  const { settings, members, ...otherAttributes } = organization;
+
+  const { data, error: usersError, mutate } = useApi<{
+    users: UserInterface[];
+  }>(`/admin/organization/${organization.id}/users`);
+  const users = data?.users || [];
```

Comment:
I'm slightly worried about scalability here, especially on Cloud.  We have pagination, but this will still fire 50 API calls at once as the code is written, each doing multiple Mongo queries.

Is this users data needed for the collapsed row?  Or can we wait until the row is expanded before making this API call?

Reply:
Made those changes here https://github.com/growthbook/growthbook/pull/2528